docs/architecture/overview.md

```diff
@@ -10,6 +10,9 @@ This document specifies the architecture for GCP Ingestion as a whole.
   Firefox) to PubSub `Raw Topics`
 - The Dataflow `Landfill Sink` job copies messages from PubSub `Raw Topics` to
   `Cloud Storage`
+- The Dataflow `EcosystemDecryptor` job reads from a `Raw Topic` specific to
```

Comment:
Would it be possible to indicate where this operates in relation to the jobs?  Ideally, updating the diagram would help, otherwise some simple textual description for when this invokes relative to (I think?) `Ingestion Edge`, `Landfill Sink`, and `Decoder` would help me.